week-2/implement/cases.js

```diff
@@ -15,3 +15,12 @@
 
 // Come up with a clear, simple name for the function
 // Use the string documentation to help you plan your solution
+
+
+function text(y){
+    return y.replaceAll(" ","_").toUpperCase();
```

Comment:
Well done on creating a succinct function that satisfies the requirements.
Think about the naming of your function and variable here - how can you make these more descriptive so that anyone reading the name can easily understand the purpose?